pandas/io/formats/printing.py

```diff
@@ -3,13 +3,14 @@
 """
 
 import sys
+from typing import Callable, Dict, Iterable, List, Optional, Sequence, Tuple, Union
 
 from pandas._config import get_option
 
 from pandas.core.dtypes.inference import is_sequence
 
 
-def adjoin(space, *lists, **kwargs):
+def adjoin(space: int, *lists: str, **kwargs) -> str:
```

Comment:
Does this mean all the lists are strings?

Reply:
nice catch. should be List[str]. will update.